test/Deposit.spec.ts

```diff
@@ -12,7 +13,7 @@ const RollupUtils = artifacts.require("RollupUtils");
 import { StateStore } from "../scripts/helpers/store";
 chai.use(chaiAsPromised);
 
-contract("DepositManager", async function(accounts) {
+contract("DepositManager", async function() {
```

Comment:
If you need to use `Contract.deployed()` use contract. It runs `truffle-fixture.js`